CHANGELOG.md

```diff
@@ -1,7 +1,22 @@
-## master
-    - Support for chunk cache configuration at data set create/open
-    - Support for std::array (unnested)
-    - Added AnnotateTraits::createAttribute and NodeTraits::createDataset overloads to create and initialize the data in a single call
+## Version 2.1 - 2019/10/XX
+### New Features:
+    - Inspection: API to get the type of links/objects and datasets data-types (#221)
+    - H5Easy: API for simple import/export to Eigen and xtensor (#141)
+    - Support for chunk and deflate configuration at dataset creation/open (#125). Added generic RawPropertyLists. (#157)
+    - Recursive `createGroup` and `exist` (#152)
+    - Shortcut syntax: ability to create a filled dataset in a single line (#130)
+    - DataSet now accepts std::complex and std::array's (#128, #129)
+
+### Improvements
+    - Improved compat with MSVC and ICC compilers
+    - Cmake build system: modernized, create exported targets, better messages, etc.
+    - Building and publishing documentation: https://bluebrain.github.io/HighFive/
```

Comment:
On https://bluebrain.github.io/HighFive : 

```
HighFive  3.0
HighFive - Header-only C++ HDF5 interface
```

3.0 ??

Reply:
It was a test to show the output. Once this patch is merged it will have the correct version number.